lib/chef/provider/aws_s3_bucket.rb

```diff
@@ -74,13 +72,27 @@ def destroy_aws_object(bucket)
 
   private
 
+  def website_exist?(new_resource,bucket)
+    return true if new_resource.driver.s3.get_bucket_website(bucket: bucket.name)
+  rescue
+    return false
+  end
+
+  def create_website(bucket,new_resource )
+    website_configuration = Aws::S3::Types::WebsiteConfiguration.new(
+            new_resource.website_options)
+    s3 = new_resource.driver.s3
+    s3.put_bucket_website( bucket: bucket.name,  website_configuration:website_configuration)
+    
```

Comment:
there's a blank line here and the whitespace in the line above this is inconsistent (remove the space between `(` and `bucket:`, I think you want to have a single space after the colon between keywords, i.e. `bucket: bucket.name` and `website_configuration: website_configuration`.